src/main/java/org/sagebionetworks/bridge/spring/controllers/SubstudyController.java

```diff
@@ -49,22 +50,22 @@ final void setSubstudyService(SubstudyService substudyService) {
     @PostMapping("/v3/substudies")
     @ResponseStatus(HttpStatus.CREATED)
     public VersionHolder createSubstudy() {
-        UserSession session = getAuthenticatedSession(ADMIN);
+        UserSession session = getAuthenticatedSession(SUPERADMIN);
```

Comment:
Previously, study admins could manage substudies. Is it intentional that only superadmins can manage substudies?

Reply:
Yes, these should be created by us (as super admins) as they represent organizations and their access to the users in the app.